app/layout.tsx

```diff
@@ -34,6 +35,7 @@ export default function RootLayout({
         <ThemeProvider attribute="class" defaultTheme="system" enableSystem>
           {children}
         </ThemeProvider>
+        <Toaster />
```

Comment:
Sorry to comment again, please move it inside theme provider, otherwise the theme won't apply to toaster